sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/DataSourceV2AnalysisSuite.scala

```diff
@@ -111,6 +115,20 @@ case class TestRelationAcceptAnySchema(output: Seq[AttributeReference])
 }
 
 abstract class DataSourceV2AnalysisSuite extends AnalysisTest {
+
+  override def getAnalyzer(caseSensitive: Boolean): Analyzer = {
+    val conf = new SQLConf()
+      .copy(SQLConf.CASE_SENSITIVE -> caseSensitive)
+      .copy(SQLConf.STORE_ASSIGNMENT_POLICY -> StoreAssignmentPolicy.STRICT)
+    val catalog = new SessionCatalog(new InMemoryCatalog, FunctionRegistry.builtin, conf)
+    catalog.createDatabase(
+      CatalogDatabase("default", "", new URI("loc"), Map.empty),
```

Comment:
how about creating a temporary dir for loc?

Reply:
This is actually copied from `AnalysisTest`. I think it should be fine.